src/Microsoft.Health.SqlServer/Features/Schema/Manager/SchemaClient.cs

```diff
@@ -27,7 +27,10 @@ public SchemaClient()
 
         public void SetUri(Uri serverUrl)
         {
-            _httpClient.BaseAddress = serverUrl;
+            if (_httpClient.BaseAddress == null)
```

Comment:
Why is this needed? If I were calling this method, I wouldn't expect that it can't be changed after the first call.

Reply:
Agree! But while I was doing some local testing, probably last week I noticed that if by chance SetUri is called while BaseAddress is already present, it throws exception. So I just wanted to ensure to set only if its null. Unfortunately, I can't repro this now and I don't remember exactly what were my local changes when I noticed this.